components/actionbar/metadata/actionbar.yml

```diff
@@ -68,19 +68,19 @@ examples:
 
           <div class="spectrum-ActionGroup spectrum-ActionGroup--sizeM">
             <button class="spectrum-ActionButton spectrum-ActionButton--sizeM spectrum-ActionButton--quiet spectrum-ActionGroup-item">
-              <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true" aria-label="Edit">
+              <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-label="Edit">
```

Comment:
Ok so it looks like you removed `aria-hidden` from the buttons that also have text in them. This causes this button to be read "Edit Edit" by the screen reader because it's reading the SVG label and the button text. So this is a place where we actually want to keep the `aria-hidden` so that doesn't happen. The buttons below this that only have icons in them (lines 103-118ish) are where you would want to remove the `aria-hidden` because they don't have button text and the screen reader can't read the aria-label because of the aria-hidden and it just reads "button"